Source/DafnyCore/Options/BoogieOptionBag.cs

```diff
@@ -11,21 +11,19 @@ public static class BoogieOptionBag {
   public static readonly Option<IEnumerable<string>> BoogieFilter = new("--boogie-filter", @"
 (experimental) Only check proofs whose Boogie name is matched by pattern <p>. This option may be specified multiple times to match multiple patterns. The pattern <p> may contain * wildcards which match any character zero or more times. If you are unsure of how Boogie names are generated, please pre- and postfix your pattern with a wildcard to enable matching on Dafny proof names."
     .TrimStart()) {
-    ArgumentHelpName = "pattern"
+    ArgumentHelpName = "pattern",
+    AllowMultipleArgumentsPerToken = true,
```

Comment:
Could you set this for all options? 

You can write something like 
```c#
    foreach (var option in Commands.SelectMany(c => c.Options)) {
        option. AllowMultipleArgumentsPerToken = true;
    }
```

In `CommandRegistry.Create`

Reply:
Made this change